src/common/utils/FocusManager.ts

```diff
@@ -83,7 +80,7 @@ export abstract class FocusManager {
     // Whenever the focusable element is mounted, we let the application
     // know so that FocusManager could account for this element during the
     // focus restriction.
-    addFocusableComponent(component: FocusableComponentInternal) {
+    addFocusableComponent(component: FocusableComponentInternal, accessibleOnly?: boolean) {
```

Comment:
Perhaps it would be better to define accessibleOnly as a required parameter but with a default value of "false"? It seems unintuitive for a parameter like this to be completely optional.

Reply:
Also, a comment would be useful explaining what "accessibleOnly" means here.